rise-storage.html

```diff
@@ -1152,7 +1143,30 @@
      */
     go: function() {
       if (this._pingReceived) {
-       this._getStorageSubscription();
+        if (!this._isCacheRunning) {
+          this._getStorageSubscription();
+        }
+        else {
+          // get file immediately from Rise Cache
+
+          if (this.companyid) {
+            if (this._isLoading) {
+              // determine if configured for a file
+              this._isFile = (this.filename !== "");
+            }
+
+            if (this._isFile) {
+              // configure the file path
+              this._fileUrl = encodeURIComponent("https://storage.googleapis.com/" + this._getStoragePath());
+
+              this._getFileFromCache();
+            }
+            // Folder
+            /*else {
+              // TODO: handle folder
```

Comment:
@stulees I think you can just call `this._getStorageSubscription();` for folder in here. cheers